system-parachains/asset-hubs/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -684,10 +698,14 @@ pub mod bridging {
 			sp_std::vec::Vec::new().into_iter()
 			.chain(to_polkadot::BridgeTable::get())
 			.collect();
+
+		pub EthereumBridgeTable: sp_std::vec::Vec<NetworkExportTableItem> = to_ethereum::BridgeTable::get();
```

Comment:
```suggestion
```

as suggested before, just to remove this (also, this location was meant to be common for all bridges and this is ETH specific)